shared/modules/persistence_module.go

```diff
@@ -143,236 +143,16 @@ type PersistenceContext interface {
 	// Params
 	InitParams() error
 
-	GetParamAppMinimumStake() (string, error)
-	GetMaxAppChains() (int, error)
-	GetBaselineAppStakeRate() (int, error)
-	GetStabilityAdjustment() (int, error)
-	GetAppUnstakingBlocks() (int, error)
-	GetAppMinimumPauseBlocks() (int, error)
-	GetAppMaxPausedBlocks() (int, error)
+	GetIntParam(paramName string, height int64) (int, error)
+	GetStringParam(paramName string, height int64) (string, error)
+	GetBytesParam(paramName string, height int64) ([]byte, error)
+	SetParam(paramName string, value interface{}) error
 
-	GetParamServiceNodeMinimumStake() (string, error)
-	GetServiceNodeMaxChains() (int, error)
-	GetServiceNodeUnstakingBlocks() (int, error)
-	GetServiceNodeMinimumPauseBlocks() (int, error)
-	GetServiceNodeMaxPausedBlocks() (int, error)
-	GetServiceNodesPerSession() (int, error)
+	// Flags
+	InitFlags() error
```

Comment:
Having a hard time finding documentation on 'flags' and why they are necessary; please add

Reply:
In the issue @deblasis had originally taken on, he actually asked about the difference (from a technical and product standpoint) between flags and params here: https://github.com/pokt-network/pocket/issues/76

I was the one who suggested that we implement the two in similar fashions, resuing a lot of code/infrastrucutre with one key difference: params are protocol level params that the drive the state of the protocol (i.e. a value such as a fee), while a flag is VERY similar but has an `enabled` flag and can, depending on the situation, determine if the value of a param is used or not but keeping the two separate.

_Perhaps we can change a flag to be a float in the future like in other feature flag systems but I think that's overkill for now_

I think we should keep it as is for now, but perhaps add a bit of documentation in a README or Changelog.

Reply:
Yeah - understood. 

Also, it might be a bit premature to add upgrade functionality for M1.

In any case, **not a blocker**. I just think it's important to have some documentation